### PR TITLE
ci: fix flaky e2e test

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Sometimes

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo-integrations/cypress/tests/kit/slider/slider.spec.ts [hunk 1]
```diff
@@ -3,22 +3,15 @@ import {
...
 
-const goToApiPage = () => {
-    cy.get('[tuitab]').eq(1).click();
-};
-
 describe('Slider', () => {
     beforeEach(() => {
         cy.viewport('macbook-13');
-        cy.goToDemoPage(SLIDER_PAGE_URL);
+        cy.goToDemoPage(`${SLIDER_PAGE_URL}/API?tuiMode=null&max=89&min=0&step=1&size=s`);
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/tests/kit/slider/slider.spec.ts:9]: will it work in our proprietary repo ?

Comment (splincode) [reply projects/demo-integrations/cypress/tests/kit/slider/slider.spec.ts:9]: Other specs in main branch also use `API`

image

image

**Review by nsbarsukov — APPROVED** [review]

**Comment by splincode** [comment]
@vladimirpotekhin hello, please review me)

**Comment by splincode** [comment]
@waterplea @vladimirpotekhin hello, please review me)

**Review by waterplea — APPROVED** [review]

**Pull request merged**